roadmap/roadmap.md

```diff
@@ -36,52 +37,81 @@ This Roadmap captures high level actionable items that we as a project aim to un
 
+Astronomers perform their work in a variety of computing environments with a variety of requirements. This category encompasses goals related to supporting the performance and hardware needs of Astropy Project users, as well as  anticipating future needs and requirements.
+
 - :large_orange_diamond: Improve and/or maintain interoperability with performant I/O file formats and libraries such as HDF5 and Dask.
 
 - :red_square: Improve support for using Astropy tools in heterogeneous computing environments such as cloud environments or GPU systems.
 
+- :red_square: Add basic pyscript/webassembly support in Astropy.
+
+
 ## Learn and User Support
 
+The Astropy Project cannot succeed in its core objectives without a robust user onboarding and support system.  This category is for items related to the Astropy Learn ecosystem and other user support related improvements.
+
 - :green_circle: Generate and ingest guides and/or a series of tutorials that demonstrate Astropy Project functionality in the context of astronomical research.
 
--  :large_orange_diamond: Update Astropy website: Rework the Team page and improve the visibility of Astropy Learn.
+-  :large_orange_diamond: Update Astropy website to improve the visibility of Astropy Learn and overall reflect the Project as a whole, not just the core package.
 
-- :large_orange_diamond: Expand and build on workshop offerings by recruiting and training more facilitators and expanding the geographic diversity of meetings where we offer workshops. 
+- :large_orange_diamond: Develop tutorials that demonstrate the use of Astropy for spectroscopy tasks.
 
-- :large_orange_diamond: Expand the number and diversity of active user support resources.
+- :red_square: Expand and build on workshop offerings, including online and increased geographic coverage.
 
-- :red_square: Develop tutorials that demonstrate the use of Astropy for spectroscopy tasks.
+- :red_square: Develop/harvest tutorials suitable for use in university astronomy courses.
 
-- :red_square: Develop/harvest tutorials suitable for use in university astronomy courses. 
 
 ## Community Building and Sustainability
 
-- :large_orange_diamond: Increase the learning and mentoring opportunities for people interested in becoming contributors and helping to develop existing contributors.
+As a commnunity driven project the long term health and development of the Astropy community is an intrinsic part of the Project. This category encompasses goals related to growing and maintining that community, and ensuring it adheres to the core principles of inclusion and consensus-building.
 
-- :large_orange_diamond: Better understand Astropy user community through a NumPy-like user survey (see [this summary](https://numpy.org/user-survey-2020-details/) of their results).
+- :green_circle: Better understand Astropy user community through a NumPy-like user survey (see [this summary](https://numpy.org/user-survey-2020-details/) of their results).

```

Comment:
Is this referring to the Mycology reports we discussed? Should we link them here?